kiwi/builder/disk.py

```diff
@@ -986,12 +987,44 @@ def _get_boot_names(self):
                 self.boot_image.boot_root_directory
             )
         if self.initrd_system and self.initrd_system == 'dracut':
+            dracut_output_format = self._get_dracut_output_file_format()
             return boot_names_type(
                 kernel_name=kernel_info.name,
-                initrd_name='initrd-' + kernel_info.version
+                initrd_name=dracut_output_format.format(
+                    kernel_version=kernel_info.version
+                )
             )
         else:
             return boot_names_type(
                 kernel_name='linux.vmx',
                 initrd_name='initrd.vmx'
             )
+
+    def _get_dracut_output_file_format(self):
+        """
+        Unfortunately the dracut initrd output file format varies between
+        the different Linux distributions. Tools like lsinitrd, and also
+        grub2 rely on the initrd output file to be in that format.
+        Thus when kiwi uses dracut the same file format should be used
+        all over the place in order to stay compatible with what the
+        distribution does
+        """
+        default_outfile_format = 'initrd-{kernel_version}'
```

Comment:
I would strongly suggest that you fallback to the default for `dracut`: `initramfs-{kernel_version}.img`, as that is more likely to work when you can't figure out what the format is.